DyamicLinking.md

```diff
@@ -0,0 +1,34 @@
+# Dynamic linking
+
+Dynamic loading of code is in [the MVP](MVP.md) in the form of
+[modules](Modules.md), but all loaded modules have their own separate
+[linear memory](AstSemantics.md#linear-memory) and cannot share
+[function pointers](ASTSemantics.md#calls). Dynamic linking will allow
+developers to share memory and function pointers between WebAssembly modules.
+
+WebAssembly will support both load-time and run-time (`dlopen`) dynamic linking
+of both WebAssembly modules and non-WebAssembly modules (e.g., on the Web, ES6
+ones containing JavaScript).
```

Comment:
Yeah, I wondered about that. We will already have module imports and exports, (so load-time linking), but in general it's probably outside the scope of this design anyway, so I'll just remove it.